assets/src/models/ladderVehicle.ts

```diff
@@ -220,9 +218,13 @@ const overlappingPreviousVehicles = (
     areOverlapping(previousVehicle, vehicle)
   )
 
+// We need to determine overlapping vehicles before knowing the height of the
```

Comment:
Why? I know we might have to start with a default height, but once the height is set, we should be able to read that value, right?

Reply:
This went away when we stopped passing height around.